backend/Agents/Citizen.py

```
@@ -0,0 +1,142 @@
+            personality=self.personality,
+            publicRecord=self.getPublicRecords(politicians),
+            posts="\n\n".join(search_results),
+            memories=self.recall()
+        )
+
+        logger.debug(f"Generated Decision Prompt: {prompt}")
+
+        # Use LLM to make a decision
+        try:
+            resObj = request_ollama(prompt).strip()
+            resObj = json.loads(resObj.strip("```json").strip("```"))
```

Ruff (0.12.2)

Using `.strip()` with multi-character strings is misleading

(B005)